SP23.P03.Web/ClientApp/src/components/Nav/Navbar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { Menu } from "semantic-ui-react"; 
+import { routes } from "../../constants/routeconfig";
+import "./NavbarStyling.css"; 
+
+export function Navbar() : React.ReactElement {
+    return (
+        <nav className="nav-container">
+
+            <Menu
+            borderless pointing
+            className="item-container">
+                
+                <Menu.Item
+                    name='EnTrack'
```

Comment:
```suggestion
                    content='EnTrack'
```
Seems like the `name` prop likes to always put spaces before capital letters for some reason. So I guess if we want the button to say "EnTrack" instead of "En Track" we have to use the `content` prop instead.